include/jsonbuilder/JsonBuilder.h

```diff
@@ -419,21 +478,23 @@ nodes.) To add new types in your own project:
   and jsonValue.ConvertTo<T> to work for your types,
   define specializations of JsonImplementType<T> for each of your types.
 */
-enum JsonType : uint8_t
+enum JsonType
+    : JsonInternal::JSON_UINT32 // Note: Used as an 8-bit bitfield of UINT32.
```

Comment:
Note: `JsonType` is used in a bitfield, and the other member of the bitfield is a `UINT32`. It is nonstandard to depend on bitfields distinct types (i.e. where one part of the bitfield is `UINT8` and the other part is `UINT32`). Change `JsonType` to have a base type of `UINT32` so that we're not depending on nonstandard behavior, i.e. so we can rely on a `JsonType` working nicely in a bitfield together with another `UINT32` field.